lib/src/includes/signed_video_sign.h

```diff
@@ -342,26 +342,13 @@ signed_video_set_end_of_stream(signed_video_t *self);
  *     // Handle error
  *   }
  *   // All necessary configurations need to be done prior to this call.
```

Comment:
Remove this line. It is wrong. The necessary configuration has to be done before `signed_video_generate_golden_sei()`